utils/line_ending_file/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "line_ending_file"
```

Comment:
Discussion: Let's pick a better name for this crate.

- `crlfify`
- `glazier` (someone who works on windows)
- `carriage_glazier` (more specific)
- `coachman` (someone who drives carriages, i.e. carriage returns)

Reply:
I don't think your suggestions are good names. They are very clever if you already know what the crate is doing, but useless otherwise.

Reply:
I'm trying to learn from @Manishearth on how to come up with clever names, like Yoke :)

The least clever of the names is `crlfify`.

I'm happy with a more boring name, too, but neither `line_ending_file` nor `text_file` are very descriptive, unless we plan to add more text-file-related utilities to this crate. I think we should limit the crate to only solving CRLF problems.

Reply:
We could remove the first `f` and make it `crlify`, which is easier to pronounce